backend/open_webui/models/auths.py

```
@@ -105,27 +108,58 @@ def insert_new_auth(
+                    return user_model
+                else:
+                    return None
 
-            if result and user:
-                return user
-            else:
-                return None
+            except Exception as e:
+                # Ensure rollback on any error
+                db.rollback()
+                log.exception(f"Error in insert_new_auth: {str(e)}")
```

Ruff (0.13.1)

Use explicit conversion flag

Replace with conversion flag

(RUF010)